web/packages/plugins/analysis-platform/src/modules/Analysis/components/AnalysisJobTable.tsx

```diff
@@ -43,6 +43,7 @@ const JobRow = (props: {
   setJob: (j: TJob) => void
 }) => {
   const { job, index, analysisId, dataSourceId, removeJob, setJob } = props
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore